core-blocks/image/index.js

```diff
@@ -217,6 +217,16 @@ export const settings = {
 			/>
 		);
 
+		if ( 'left' === align || 'right' === align ) {
```

Comment:
We will need a deprecation handler for this change. This will make all old blocks having `align` set to right or left to be recognized as externally modified.